Automattic/newspack-plugin
# Refactor/wizards

### All Submissions:






### Changes proposed in this Pull Request:



Refactors Support & Syndication Wizards to use the new `Wizard` component.

### How to test the changes in this Pull Request:

Visit Support & Syndication wizards, do a smoke test to verify they work as before

### Other information:
